### ISSUE TITLE
Settings window opens behind other windows sometimes.

### ISSUE THREAD

**Issue by blackketter** [lukepistrol/TimeMachineStatus#16]
I noticed that if I have a Safari window frontmost and choose "Settings" from the TimeMachineStatus popup menu, it may open behind the window and not be visible. 

I've also seen that if the settings window is open already and I choose "Settings" it doesn't appear to be made frontmost.

**Comment by lukepistrol** [comment]
Right. Noticed that too. Unfortunately this seems to be an issue with `SettingsLink`. If anyone knows a workaround that will work on macOS 14 please join the discussion!
